src/HotCorners/HotCorner.vala

```diff
@@ -38,25 +38,57 @@ public class Gala.HotCorner : Object {
      * In order to avoid accidental triggers, don't trigger the hot corner until
      * this threshold is reached.
      */
+    private const int TRIGGER_PRESSURE_THRESHOLD_MIN = 0;
     private const int TRIGGER_PRESSURE_THRESHOLD = 50;
 
     /**
      * When the mouse pointer pressures the barrier without activating the hot corner,
      * release it when this threshold is reached.
      */
-    private const int RELEASE_PRESSURE_THRESHOLD = 100;
+    private const int RELEASE_PRESSURE_THRESHOLD_MIN = 30;
```

Comment:
This threshold is mainly used when using more than one screen. The barrier catches the mouse a bit and allows to move it to the next screen if the user keeps pressing the corner. I don't know if we need to change it.

Reply:
Ok